.circleci/config.yml

```diff
@@ -1438,6 +1396,7 @@ workflows:
             - op-program-docker-build
             - op-supervisor-docker-build
             - proofs-tools-docker-build
+            - go-test-all
```

Comment:
This would be a good one to add to the list of explicitly required tasks in GitHub (maybe in a few days after PRs have had a chance to update or be merged before blocking on it).  If it's not specifically listed the merge queue takes longer to kick things out when it fails.

Although surprisingly none of the old ones were explicitly required so could just see how it goes.

Reply:
Good idea - will do so once the current batch of PRs goes through.